x-pack/test_serverless/functional/test_suites/common/management/data_views/_data_view_create_delete.ts

```diff
@@ -53,7 +58,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.settings.clickKibanaIndexPatterns();
         await PageObjects.settings.clickAddNewIndexPatternButton();
         await testSubjects.click('closeFlyoutButton');
-        await testSubjects.find('createDataViewButton');
```

Comment:
Can you help me understand why this line was removed? To me it looks like kind of the assertion in this test so I'm wondering if the test still makes enough sense without it :thinking:

Reply:
@pheyos Thats a good point, I overlooked that. I'm restoring this line and making the test work even if there's an existing data view. The test was failing because there was an existing one but the test isn't really checking for that.